hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -3607,4 +3607,12 @@
       without using the optimised DAG based pruning approach
     </description>
   </property>
+  <property>
+    <name>ozone.scm.check.empty.container.delete</name>
+    <value>true</value>
+    <tag>DATANODE</tag>
```

Comment:
`ozone.scm.` prefix doesn't seem to be the best one for datanode-specific setting.

Reply:
I was debating if this is something SCM cares about, but Datanodes verify of just a local datanode check. Will move it.